examples/rvc-app/rvc-common/rvc-app.zap

```diff
@@ -1426,6 +1426,14 @@
               "isIncoming": 1,
               "isEnabled": 1
             },
+            {
```

Comment:
Oh, I see now! I glossed over this and didn't realize the implication. Good find!

Reply:
I think I missed the earlier discussion here, but if this command was included here because the feature map is set such that this is required, i would suggest removing the feature rather than implementing this command unless you're sure it's required. 

See ThreatModel doc - T154, CM154 - basic commissining window is less secure than ECM and unless there's a good reason to have it, it's best to remove it from the examples lest it slip into products.
(https://groups.csa-iot.org/wg/matter-tsg/document/17911)

Reply:
Not sure I understand what you are talking about. This command was accidentally removed due to some zap tool issues which meant that we were not able to use chip-tool to open the commissioning window as we used to be able to do. This is required to allow both chip-tool and the tests to have access to the app.